compile.sh

```diff
@@ -335,7 +335,10 @@ echo "}" >> test.c
 
 type $CC >> "$DIR/install.log" 2>&1 || { echo >&2 "[ERROR] Please install \"$CC\""; exit 1; }
 
-[ -z "$THREADS" ] && THREADS=1;
+if [ -z "$THREADS" ]; then
+	write_out "WARNING" "Threads is not set, please set it with -j option to build faster"	
```

Comment:
```suggestion
	write_out "WARNING" "Only 1 thread is used by default. Increase thread count using -j (e.g. -j 4) to compile faster."	
```
Perhaps we should suggest a thread count based on the number of available cores, but I don't know a cross-platform way to do that.